scripts/kcc

```diff
@@ -41,6 +41,16 @@ if ($cygwin) {
   chop($distDirectory);
   chop($profileDirectory);
 }
+
+my $preprocessor;
+{
+  local $/=undef;
+  open FILE, "$profileDirectory/pp" or die "Couldn't find current profile: run kcc-switch";
+  $preprocessor = <FILE>;
+  close FILE;
+  chop($preprocessor);
+}
+
```

Comment:
I will probably do that eventually, honestly, but until we have profiles that use non-gcc preprocessors, it will be less code duplication this way. Eventually I want to have profiles be configured from individual components, so that that code duplication wouldn't happen, but that's more work to deal with right now.